contents/handbook/who-we-build-for.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Who we build for
+sidebar: Handbook
+showTitle: true
+---
+
+AKA our [ideal customer profile](/newsletter/ideal-customer-profile-framework) or ICP (ie, the company), and our persona (ie, the person).
+
+We build for the people building the **most-loved products** at **high-growth startups**. 
+
+Marketing and customer success should primarily focus on this ICP, but should also develop **high-*potential* customers** – customers that are likely to later become high-growth customers (e.g. PostHog itself during YC). We should be in maintenance mode for **hobbyists**, such as engineers building side projects. We want to be the first tool that technical founders add to their product.
+
+Because we have a lot of products, we support a few different _personas_, depending on the stage of the company (see [our current persona](#our-current-persona) section below).
+
+## Our current ICP
+
+| &nbsp; | High-growth startup |
+| --- | --- |
+| **Description** | Startups that have product-market fit and are quickly scaling up with new customers, hiring, and adding more revenue. |
+| **Criteria** | - 15-500 employees<br />- $100k+/month in revenue _or_ very large number of consumer users<br />- Raised from leading investors<br />- Not yet IPO'ed |
+| **Why they matter?** | - Able to efficiently monetize them<br />- Very quick sales cycle<br />- Act as key opinion leaders for earlier-stage startups/slower moving companies<br />- Strong opinions on what they need - helping us build a better product |
+| **Examples** | PostHog anytime from their Series B to IPO, Supabase, 11Labs |
+
+Each team will focus more or less on different members of the product team. This is detailed on their team pages.
+
+## Our current Persona
+
+During the life of PostHog, we've tried to build different types of tools for different types of people (or, personas). We've found that, as companies get bigger, the type of person that uses a product changes. As an example:
+
+- We built product analytics for engineers at startups
+- As those companies get a little bit bigger, they'll hire Product Managers who will mostly use product analytics. PMs have more complicated requirements for what a product analytics tool needs to do
+- Even bigger companies often have specialised "analytics engineers". These people are the most demanding.
+
+Some of our products will have multiple personas. It's up to each product team to decide who those are, and how to prioritise between them. 
+
+We've not always been successful at building products for personas other than engineers. We're now at a stage where we need to be in order to continue growing. 
```

Comment:
```suggestion
```

This is probably a worthwhile comment in comms about this but not sure if it adds much to this doc.

Reply:
I think it adds context _now_ but not for eg new people who are joining in the future?

Reply:
I think it's very honest and introspective. I appreciate that we have it here. Being open about our shortcomings is how we get better.

We can always remove this once we declare victory on buiilding products for non-engineering types